Dockerfile

```diff
@@ -0,0 +1,40 @@
+#
+# This Dockerfile builds Experimental Remoting image for jenkins4eval/remoting
+# This image is similar to jenkins/slave, so extra tweaks need to be applied for better UX
+#
```

Comment:
I'm not clear on exactly what this Dockerfile is used for or how it relates to the jenkins/slave one. This looks like a generic Dockerfile, which will build an image from the current local source. I'm confused about the comment regarding how this is for "jenkins4eval/remoting". Maybe it's just the comment that is confusing me and this can be used to build an image for any current version. Basically the same as jenkinsci/docker-slave, except it builds from local source.

Should these two Dockerfiles be in the same space or otherwise better connected to each other?